### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ina-stoyanova**
Add the bit for how to choose a release tag.

### CODE REVIEW

**Comment by infraredgirl** [comment]
I think we should also link to the [Making a New Release section](https://www.notion.so/gruntwork/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e#08b68ee0e19143e89523dcf483d2bf48) of the Coding Methodology RFC.

**Comment by ina-stoyanova** [comment]
Oh thanks, Ana! The email update for this comment must have been lost somewhere in my inbox, so apologies for the delay! I think that's a good suggestion!

**Comment by ina-stoyanova** [comment]
Done ✅  Do you mind giving this another look @infraredgirl?

File: README.md [hunk 1]
```diff
@@ -350,9 +350,15 @@ TEST_ACMPCA_EXPENSIVE_ENABLE=1 go test -v ./...
 Every source file in this project should be formatted with `go fmt`.
 
 ### Releasing new versions
+We try to follow the release process as deifned in our [Coding Methodology](https://www.notion.so/gruntwork/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e#08b68ee0e19143e89523dcf483d2bf48). _This document is only visible to Gruntwork employees currently._
```
Comment (infraredgirl) [inline README.md:353]: The coding methodology document is actually externally visible, as it says on the top of that Notion page.

Comment (ina-stoyanova) [reply README.md:353]: Oh good point! I saw the settings on the page, but interpreted them wrong!

**Review by infraredgirl — APPROVED** [review]
LGTM

**Comment by ina-stoyanova** [comment]
Going to merge as is: 

Failing tests have nothing to do with this PR. There's already an open issue for fixing the failing tests: https://github.com/gruntwork-io/cloud-nuke/issues/227

The failing tests:
Screenshot 2021-11-09 at 13 48 58

**Pull request merged**